configure

```diff
@@ -7084,6 +7088,22 @@ else
 fi
 define_in_config_if_yes $_taskbar 'USE_TASKBAR'
 
+#
+# Check whether to build printing support
+#
+echo_n "Building system printing support... "
+if test "$_printing" = "no"; then
+	echo "no"
```

Comment:
I think that you may actually always have a fallback that will save printouts to images.